README.md

```diff
@@ -62,18 +62,25 @@ This package assumes that you use Intercom's `company tag`, `contact tag`, `cont
 
 ...
 vars:
+  using_contact_company: False
```

Comment:
should we prefix these with `intercom_` if they're global?

Reply:
Updated the variables to be in line with our global naming convention.